### PR TITLE
[#1] Generate project from flutter-templates

### PR DESCRIPTION
**Pull request by Tuubz**
Closes https://github.com/nimblehq/ic-flutter-taher-toby/issues/1

## What happened 👀

- Generate project from flutter-templates
- Update CODEOWNERS
- Update GitHub templates
- Update README.md
- Remove `story_template.md`

## Proof Of Work 📹

App can be built successfully: 

Screen Shot 2023-03-20 at 10 36 15

### CODE REVIEW

**Comment by Tuubz** [comment]
If I'm not mistaken, the Codecov warning is showing because there's currently no data in `develop` to compare with

**Review by Shayokh144 — APPROVED** [review]
looks good to me.

File: scripts/setup.py
Comment (luongvo) [inline scripts/setup.py:1]: We don't need this script in the generated project.

File: scripts/test.py
Comment (luongvo) [inline scripts/test.py:1]: same

**Review by luongvo — CHANGES_REQUESTED** [review]
The rest lgtm

File: scripts/test.py
Comment (Tuubz) [reply scripts/test.py:1]: @luongvo Updated 🚀

**Review by luongvo — APPROVED** [review]

**Pull request merged**